locales/locales.php

```diff
@@ -1016,6 +1044,15 @@ public function __construct() {
 		$fo->slug = 'fo';
 		$fo->facebook_locale = 'fo_FO';
 
+		$fon = new GP_Locale();
+		$fon->english_name = 'Fon';
+		$fon->native_name = 'fɔ̀ngbè';
+		$fon->lang_code_iso_639_2 = 'fon';
+		$fon->lang_code_iso_639_3 = 'fon';
+		$fon->country_code = 'BJ';
+		$fon->wp_locale = 'fon_bj';
```

Comment:
```suggestion
		$fon->wp_locale = 'fon';
```

Three-letter codes only need the country code if we really have a second request for the same language.